### PR TITLE
Convert huggingface model to onnx

### PR DESCRIPTION
**Pull request by xyang16**
## Description ##

Brief description of what this PR is about

- If this change is a backward incompatible change, why must this change be made?
- Interesting edge cases to note here

### CODE REVIEW

**Review by lanking520 — COMMENTED** [review]
I think it is worthwhile for us to build a package for this onnx model conversion and standardize the UX

File: serving/docker/convert/.gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+__pycache__
```
Comment (frankfliu) [inline serving/docker/convert/.gitignore:1]: We don't need this file

**Comment by xyang16** [comment]
> I think it is worthwhile for us to build a package for this onnx model conversion and standardize the UX

Done

File: serving/docker/lmi.Dockerfile [hunk 2]
```diff
@@ -92,7 +93,9 @@ RUN pip3 install torch==${torch_version} torchvision==${torch_vision_version} --
     transformers==${transformers_version} hf-transfer zstandard datasets==${datasets_version} \
     mpi4py sentencepiece tiktoken blobfile einops accelerate==${accelerate_version} bitsandbytes==${bitsandbytes_version} \
     optimum==${optimum_version} auto-gptq==${auto_gptq_version} pandas pyarrow jinja2 \
-    opencv-contrib-python-headless safetensors scipy && \
+    opencv-contrib-python-headless safetensors scipy \
+    onnxruntime-gpu==${onnx_version} --extra-index-url https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/onnxruntime-cuda-12/pypi/simple/ \
```
Comment (frankfliu) [inline serving/docker/lmi.Dockerfile:97]: Why we need onnxruntime-gpu?, we only use it for conversion

**Review by frankfliu — APPROVED** [review]

**Pull request merged**